IRremoteInt.h

```diff
@@ -211,6 +211,7 @@ typedef struct {
   unsigned int timer;     // state timer, counts 50uS ticks.
   unsigned int rawbuf[RAWBUF]; // raw data
   uint8_t rawlen;         // counter of entries in rawbuf
```

Comment:
the change is not part of this commit so it shouldn't be here.
also currently rawbuf is 100 so uint8 is ok. maybe adding a comment next to rawbuf to mention rawlen type should be changed accordingly.